src/tools/Reporting/FailureReporting/FailureReporter.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Reporting;
+using System;
+using System.IO;
+
+namespace FailureReporting
+{
+    public class FailureReporter
+    {
+        public static void Main(string[] args)
+        {
+            if (args.Length == 0)
+            {
+                throw new ArgumentException("FailureReporting.exe was called without the required JSON path argument");
+            }
+            else if (args.Length > 1)
+            {
+                throw new ArgumentException("FailureReporting.exe was called with more than one argument");
+            }
+            else if (!Directory.Exists(Path.GetDirectoryName(args[0]))) {
```

Comment:
You can just use `File.Exists` here instead of `Directory.Exists`

Reply:
The argument is a directory name, since I need to know whether the directory exists, and the json file hasn't been created yet.  Are you sure File.Exists is correct for this?

Reply:
Ah. This bit of  your change made me think you were passing in the location of an existing json file and not making a new one. My mistake.

https://github.com/dotnet/performance/pull/1703/files#diff-55f4d5691c43ac42f361039296fae993f269730c60f83fbd7c5aa103d3892879R241

In that case the directory passed in to create the file should be the helix upload directory. Look in `util.py` in the scenarios folder for a pointer there, and again you may want to refactor that code into `util.py`.